can/interface.py

```diff
@@ -20,6 +20,8 @@
 from .util import load_config
 from .interfaces import BACKENDS
 
+from can.interfaces.socketcan.socketcan import CyclicSendTask, MultiRateCyclicSendTask
```

Comment:
The module name `socketcan.socketcan` is a bit redundant.

Reply:
Agree. Do you have a suggestion for a new name?

Reply:
Uhm; `impl`? Not really better though ...

Reply:
Why not just hoist them into `can.interfaces.socketcan`? 
In `can.interfaces.socketcan.__init__` we can import the classes and functions that make up the implementation

Reply:
Do you mean that we should make the current socketcan.py __init__.py?

Reply:
Nah (although that would work), I was just more commenting that no matter what file the classes end up in we should expose them via `can.interfaces.socketcan` e.g., `can.interfaces.socketcan.CyclicSendTask`.

Reply:
That’s already done so this is more for internal architecture.